charts/argo/values.yaml

```diff
@@ -33,7 +33,8 @@ controller:
     enabled: false
     path: /metrics
     port: 8080
-  persistence: {}
+  persistence:
+    archive: false
```

Comment:
Is the point here just to document this is an option? if so default is already false? perhaps we can just add it as a comment to document the option.

Reply:
@stefansedich you're right. A commented out version of this would be just fine. I will make this change.

Reply:
@stefansedich moved to a comment in the latest commit.

Reply:
Thanks! mind just rebasing and bumping the chart version to the next available?

Reply:
@stefansedich turns out the exact same issue was raised in this [accepted PR](https://github.com/argoproj/argo-helm/pull/363). So there's no reason to merge this. Thanks for reviewing this tho. I'm closing this MR.

Reply:
Oops my bad I should have remembered this considering I merged that one :laughing: thanks again!